xbmc/cores/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -328,12 +328,27 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
 {
   CRect src, dst;
   g_renderManager.GetVideoRect(src, dst);
-
-  int width  = MathUtils::round_int(dst.Width());
-  int height = MathUtils::round_int(dst.Height());
-
+  RESOLUTION_INFO res = g_graphicsContext.GetResInfo();
+
+  int frameWidth = res.iWidth;
```

Comment:
the term "frame" is misleading. please choose something more meaningful and add some comments which describe the parameters